pyshorteners/shorteners/cuttly.py

```
@@ -10,32 +10,31 @@ class Shortener(BaseShortener):
     Params Needed :API key
 
     
-    They too provide analyitcs for the URL 
+    They too provide analytics for the URL 
     that you shortened Features to get analytics will be implemented when the author
     approves this PR
     """
 
     api_url = 'https://cutt.ly/api/api.php '
 
-    def short(self, url, api_key=None):
-            if api_key is None:
-                return "Please provide a Valid API key"
+    def short(self, url):
+            if self.api_key is None:
+                return BadAPIResponseException("Please provide a valid API Key")
```

should call `raise` instead of `return`